railties/lib/rails/commands/encrypted/encrypted_command.rb

```diff
@@ -72,6 +72,14 @@ def encrypted_file_generator
 
           Rails::Generators::EncryptedFileGenerator.new
         end
+
+        def missing_encrypted_message(key:, file_path:)
+          if key.nil?
+            "Missing '#{options[:key]}' to decrypt data. See bin/rails encrypted:help"
```

Comment:
Mixing `key` and `options[:key]` is confusing. What's one or the other? Could renaming the `key` improve the clarity?

Reply:
`options[:key]` stores key path and `key` stores key for data decrypt. So I renamed `options[:key]` to `key_path`. What do you think?